ocs_ci/helpers/ocs_metrics_exporter_helpers.py

```
@@ -0,0 +1,1045 @@
+        pool (str): Pool name
+        image (str): Image name
+
+    Returns:
+        list: List of watcher addresses
+    """
+    cmd = f"rbd status {pool}/{image}"
+    output = exec_ceph_command(toolbox_pod, cmd)
+    watchers = []
+    for line in output.splitlines():
+        if "watcher=" in line:
+            watchers.append(line.strip())
```

Ruff (0.15.21)

[warning] Use a list comprehension to create a transformed list

(PERF401)